authentication/views.py

```diff
@@ -33,12 +33,13 @@ def _keycloak_logout_url(self, user):
             user, provider=OlOpenIdConnectAuth.name
         ).first()
         id_token = user_social_auth_record.extra_data.get("id_token")
+        qs_next = self.request.GET.get("next")
```

Comment:
You can use social auth's redirect sanitization for this: https://github.com/python-social-auth/social-core/blob/master/social_core/actions.py#L27-L31. You'll need to pull the value of allowed redirects more directly from the settings though.